app/forms/steps/income/client_owns_property_form.rb

```diff
@@ -2,7 +2,7 @@ module Steps
   module Income
     class ClientOwnsPropertyForm < Steps::BaseFormObject
       include Steps::HasOneAssociation
-      has_one_association :income_details
+      has_one_association :income
```

Comment:
Whats the reason to change the name to `income` rather than `income_details`?

Reply:
I put the explanation on one the commit in which I changed it and it got lost, sorry!

> To maintain consistency of convention in Apply, 'details' naming is
> used for the json object that is used to build the ruby object
> defined in the model of the same name without the 'details' suffix.